lib/googleDrive/index.js

```diff
@@ -0,0 +1,185 @@
+import google from 'googleapis';
+import multiparty from 'multiparty';
+import googleAuth from './googleAuth';
+import {
+  FieldValidationError,
+} from '../tools';
+
+let drive;
+
+export function setup(googleApi = {
```

Comment:
the purpose - do not forget what fields are needed) ofcause the place of defaults is in jsdoc